soc/renesas/rcar/rcar_gen3/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_SOC_SERIES_RCAR_GEN3_R7)
+  zephyr_sources(r7/soc.c)
+  zephyr_include_directories(r7)
+  zephyr_library_sources_ifdef(CONFIG_SOC_R8A77951_R7 pfc_r8a77951.c)
+
+  set(SOC_LINKER_SCRIPT ${ZEPHYR_BASE}/include/zephyr/arch/arm/cortex_a_r/scripts/linker.ld CACHE INTERNAL "")
+elseif(CONFIG_SOC_SERIES_RCAR_GEN3_A57)
+  zephyr_include_directories(a57)
+  zephyr_library_sources_ifdef(CONFIG_SOC_R8A77951_A57 pfc_r8a77951.c)
```

Comment:
note: these files will be moved in scoope of this pr https://github.com/zephyrproject-rtos/zephyr/pull/67357

Reply:
That PR will need to be rebased on this as hwmv2 will be merged after 3.6 has been complete prior to anything else and only doc/bug fixes are accepted for 3.6 now